Task 3: create hierarchy of animals

MaluLePoulet opened the pull request:
created abstract class Animal with fields: id, name and method to eat. 
created abstract classes: Bird, Feline which extend the above mentioned class.
created classes: Cat, Lion, Raven, Stork, Whale, Fish which extend the above mentioned classes. 
created interface SwimmingAnimals with method swim. 
created MainAnimals class to see the realization of classes and intefaces mentioned above.

MaluLePoulet committed:
Task 3: create hierarchy of animals
created abstract class Animal with fields: id, name and method to eat.
created abstract classes: Bird, Feline which extend the above mentioned class.
created classes: Cat, Lion, Raven, Stork, Whale, Fish which extend the above mentioned classes.
created interface SwimmingAnimals with method swim.
created MainAnimals class to see the realization of classes and intefaces mentioned above.

dmitars commented on src/Animals/Animal.java:
```diff
@@ -0,0 +1,32 @@
+        return id;
+    }
+
+    public Animal() {}
```
Seems this constructor is useless and harmful. It can create animals without ids

dmitars commented on src/MainAnimals.java:
```diff
@@ -0,0 +1,150 @@
+    private static Animal getNextAnimal() {
+        Random random = new Random();
+        int randomValue = random.nextInt(6);
+        if (randomValue == 0) {
```
How about enum?

MaluLePoulet committed:
Task 3 from homework 8.
Created enum class AnimalTypes
Edited MainAnimals class with relization of new enum class

MaluLePoulet committed:
Removed empty constructor in abstract class Animal
Edited all child classes

MaluLePoulet committed:
Homework 9 Task 1.
added 2 exception classes:
1.InvalidArraySizeException
2.InvalidUserChoiceException
modified MainAnimals class to see thrown exceptions
